onnxruntime/core/mlas/lib/wasm/sgemmv.cpp

```diff
@@ -0,0 +1,171 @@
+/*++
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    sgemmc.cpp
+
+Abstract:
+
+    This module implements the kernels for the single precision matrix/matrix
+    multiply operation (SGEMM).
+
+--*/
+
+#include "mlasi.h"
+
+size_t
```

Comment:
whoops, can you change the typedef for MLAS_GEMV_FLOAT_KERNEL in mlasi.h to be void?

Reply:
done